src/pathways/pathway/pathways.service.ts

```
@@ -1640,4 +1641,131 @@ export class PathwaysService {
+        'history.is_active',
+        'history.activated_at',
+        'user.userId',
+        'user.firstName',
+        'user.lastName',
+        'user.email',
+        'user.gender',
+      ]);
+
+      // Apply Sorting
+      if (sort?.column) {
+        const order = sort.order || 'DESC';
```

ESLint

[error] Replace `'DESC'` with `"DESC"`

(prettier/prettier)